packages/stark-ui/src/modules/app-logo/components/app-logo.component.html

```diff
@@ -1,3 +1,11 @@
-<a (click)="logoClickHandler($event)" href="#">
+<!--<a (click)="logoClickHandler($event)" href="#">
 	<i></i>
-</a>
+</a>-->
+
+<div fxLayout='row wrap' fxLayoutGap="10px">
```

Comment:
Why changing this component ?
This component is an app-logo, not a demo component for buttons... 😕 
Could you please create another component for showing those buttons ?